frontend/components/SearchBar.tsx

```diff
@@ -340,6 +341,10 @@ export const SearchBarTextItem = ({
   const inputRef = useRef() as React.MutableRefObject<HTMLInputElement>
 
   useEffect(() => {
+    doApiRequest('/searches/', {
```

Comment:
You want to put this inside the timeout logic. This will greatly reduce the number of incomplete searches that get sent to the server.

Reply:
Ok that's a great point